include/ppx/knob.h

```diff
@@ -124,7 +126,7 @@ class KnobSlider final
     static_assert(std::is_same_v<T, int>, "KnobSlider must be created with type: int");
 
     KnobSlider(const std::string& flagName, T defaultValue, T minValue, T maxValue)
```

Comment:
Not directly relevant to this PR, but what was the rational for naming the knob classes with the UI name, instead of the value they contain (KnobBoolean, KnobRange, KnobOptions)? Do you there is a use case for them to be drawn differently? We could always add  a parameter used in the `Draw` function if that is the case. 
Just thought of this, cause now the knobs can be invisible too